packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -341,11 +346,14 @@ async function getPages(
     let result = []
 
     const getOptions = page => {
+      // Do not filter tags & categories
+      let qs = url.match(/(tags|categories)/ig) ? {} : _queryParams
```

Comment:
Can you give context on this?

Reply:
Since tags and categories could be referenced across various posts even different languages filtering them could cause the gatsby-source-wordpress plugin to fail when it normalizes ids of tags&categories within posts and pages.
It could be updated to also be an option to decide exactly which routes queryParams should not be included on.

Reply:
I could see language filtering not being only use case for `queryParams`. It also depends on which multilanguage plugin (or even what settings of plugin) you use.

I would need to lookup what are potential use cases that users would want to use queryParams other than language filtering. What I'm thinking right now that `queryParams` could accept functions:
```js
// in gatsby-config
{
  resolve: `gatsby-source-wordpress`,
  options: {
    [...]
    queryParams: ({ url }) => url.match(/(tags|categories)/ig) ? {} : _queryParams
  }
}
```

Reply:
Agreed that is much more flexible.